fastlane/metadata/i18ncli.js

```diff
@@ -0,0 +1,320 @@
+/* eslint no-console: 0 */
+
+const yargs = require( "yargs" );
+const fsp = require( "fs/promises" );
+const path = require( "path" );
+
+const SUPPORTED_IOS_METADATA_LOCALES = [
+  "ar-SA",
+  "ca",
+  "cs",
+  "da",
+  "de-DE",
+  "el",
+  "en-AU",
+  "en-CA",
+  "en-GB",
+  "en-US",
+  "es-ES",
+  "es-MX",
+  "fi",
+  "fr-CA",
+  "fr-FR",
+  "he",
+  "hi",
+  "hr",
+  "hu",
+  "id",
+  "it",
+  "ja",
+  "ko",
+  "ms",
+  "nl-NL",
+  "no",
+  "pl",
+  "pt-BR",
+  "pt-PT",
+  "ro",
+  "ru",
+  "sk",
+  "sv",
+  "th",
+  "tr",
+  "uk",
+  "vi",
+  "zh-Hans",
+  "zh-Hant",
+  "appleTV",
+  "iMessage",
+  "default"
+];
+
+const IOS_MAPPINGS = {
+  ar: "ar-SA",
+  de: "de-DE",
+  fr: "fr-FR",
+  en_AU: "en-AU",
+  en_CA: "en-CA",
+  en_GB: "en-GB",
+  en: "en-US",
+  es: "es-ES",
+  es_MX: "es-MX",
+  fr_CA: "fr-CA",
+  nl: "nl-NL",
+  pt: "pt-PT",
+  pt_BR: "pt-BR"
+};
+
+const SUPPORTED_ANDROID_METADATA_LOCALES = [
+  "af",
+  "sq",
+  "am",
+  "ar",
+  "hy-AM",
+  "az-AZ",
+  "bn-BD",
+  "eu-ES",
+  "be",
+  "bg",
+  "my-MM",
+  "ca",
+  "zh-HK",
+  "zh-CN",
+  "zh-TW",
+  "hr",
+  "cs-CZ",
+  "da-DK",
+  "nl-NL",
+  "en-AU",
+  "en-CA",
+  "en-GB",
+  "en-IN",
+  "en-SG",
+  "en-US",
+  "en-ZA",
+  "et",
+  "fil",
+  "fi-FI",
+  "fr-CA",
+  "fr-FR",
+  "gl-ES",
+  "ka-GE",
+  "de-DE",
+  "el-GR",
+  "gu",
+  "iw-IL",
+  "hi-IN",
+  "hu-HU",
+  "is-IS",
+  "id",
+  "it-IT",
+  "ja-JP",
+  "kn-IN",
+  "kk",
+  "km-KH",
+  "ko-KR",
+  "ky-KG",
+  "lo-LA",
+  "lv",
+  "lt",
+  "mk-MK",
+  "ms-MY",
+  "ms",
+  "ml-IN",
+  "mr-IN",
+  "mn-MN",
+  "ne-NP",
+  "no-NO",
+  "fa",
+  "fa-AE",
+  "fa-AF",
+  "fa-IR",
+  "pl-PL",
+  "pt-BR",
+  "pt-PT",
+  "pa",
+  "ro",
+  "rm",
+  "ru-RU",
+  "sr",
+  "si-LK",
+  "sk",
+  "sl",
+  "es-419",
+  "es-ES",
+  "es-US",
+  "sw",
+  "sv-SE",
+  "ta-IN",
+  "te-IN",
+  "th",
+  "tr-TR",
+  "uk",
+  "ur",
+  "vi",
+  "zu"
+];
```

Comment:
Same here. Presumably these are locales the Play Store understands. If we were to distribute to f-droid, we'd still be distributing an Android app, but maybe there would be different locales in different formats.

Reply:
Correct, have renamed